contract/src/start-game1-proposal.js

```diff
@@ -18,75 +18,89 @@ const CENT = IST_UNIT / 100n;
 /**
  * Make a storage node for auxilliary data for a value on the board.
  *
- * @param {ERef<StorageNode>} chainStorage
- * @param {string} boardId
+ * @param {ERef<StorageNode>} chainStorage - Reference to the chain storage.
+ * @param {string} boardId - Identifier for the board.
  */
 const makeBoardAuxNode = async (chainStorage, boardId) => {
+  // Create a child node under BOARD_AUX in chain storage.
   const boardAux = E(chainStorage).makeChildNode(BOARD_AUX);
+  // Create a child node under the previously created node using boardId.
   return E(boardAux).makeChildNode(boardId);
 };
 
+/**
+ * Publish brand information for a given board and brand.
+ *
+ * @param {ERef<StorageNode>} chainStorage - Reference to the chain storage.
+ * @param {ERef<Board>} board - Reference to the board.
+ * @param {Brand} brand - Brand to be published.
+ */
 const publishBrandInfo = async (chainStorage, board, brand) => {
+  // Get the id and displayInfo of the brand.
   const [id, displayInfo] = await Promise.all([
     E(board).getId(brand),
     E(brand).getDisplayInfo(),
   ]);
+  // Create a board aux node for the brand's id.
   const node = makeBoardAuxNode(chainStorage, id);
+  // Marshal the displayInfo and set it as the value of the board aux node.
   const aux = marshalData.toCapData(harden({ displayInfo }));
   await E(node).setValue(JSON.stringify(aux));
 };
 
 /**
- * Core eval script to start contract
+ * Core eval script to start the game contract.
```

Comment:
We're trying to remove "game" from this dapp, I believe.

Reply:
"Application" instead of game?

Reply:
maybe...

but then we'd have to say what an application is... recall "dapp" vs "application" discussion

removing "game" should probably be a separate exercise. but it should perhaps go before this one :)